pkg/scheduler/actions/enqueue/enqueue.go

```diff
@@ -45,7 +46,7 @@ func (enqueue *Action) Execute(ssn *framework.Session) {
 	defer klog.V(3).Infof("Leaving Enqueue ...")
 
 	queues := util.NewPriorityQueue(ssn.QueueOrderFn)
-	queueMap := map[api.QueueID]*api.QueueInfo{}
+	queueMap := sets.NewString()
```

Comment:
If you use `sets` instead of `map`, then the name should also be `sets`

Reply:
/LGTM , thanks for your suggestion, name has fixed